pkg/api/validation/schema_test.go

```diff
@@ -39,17 +43,82 @@ func readPod(filename string) ([]byte, error) {
 }
 
 func readSwaggerFile() ([]byte, error) {
+	return readSwaggerApiFile(testapi.Default)
+}
+
+func readSwaggerApiFile(group testapi.TestGroup) ([]byte, error) {
 	// TODO this path is broken
```

Comment:
No idea.  AFAICT the path is in-fact, not broken.  If you prefer I can delete it.